Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM python:3-alpine
+
+RUN pip install fcli
+
+VOLUME ["/root/.fcli"]
```

Comment:
What's the intent here?  I could see it being useful to map the current user's `~/.fcli` into there, but I'm not sure that's what this does.